### DISCUSSION TITLE
PDF Viewer -> navigate to page in already opened PDF

### DISCUSSION THREAD

**Discussion by JochenFerchland** [cefsharp/CefSharp#4505] [General]
Hi,

it is possible to use this control to view PDF files and open them on a specific page.
It would be nice if this control would pass a goto command to a pdf to navigat to a specific page without re-opening the PDF.

**Comment by aimcosoftware** [comment]
add `#page=2` to the url

**Comment by JochenFerchland** [comment]
Hi,
thanks for your reply.
I open the pdf already with "file:///D:/test.pdf#page=5".
When I now change or add "#page=5" to "#page=2" it will reload the PDF. But I just want to navigate to another page on an already opened pdf document.

**Reply by aimcosoftware** [reply]
The only other way I know is to use pdf.js, which would give you full control.

**Reply by aimcosoftware** [reply]
There's a Wiki page now, for embedding PDF.JS [here](https://github.com/cefsharp/CefSharp/wiki/Embedding-PDF.JS-in-CEFSharp).
